### PR TITLE
Make username editable.

### PR DESCRIPTION
**Pull request by akatsoulas**
* Fix 500 error from breadcrumbs.
* View profile UI updates
* Edit profile UI updates.
* Update help_text in user form.

### CODE REVIEW

File: kitsune/sumo/static/sumo/scss/base/forms/_fields.scss [hunk 1]
```diff
@@ -355,6 +356,7 @@ textarea {
 }
 
 .delete-account-link {
+  float: right;
```
Comment (akatsoulas) [inline kitsune/sumo/static/sumo/scss/base/forms/_fields.scss:359]: This is the only change in the file.

File: kitsune/users/jinja2/users/profile.html [hunk 2]
```diff
@@ -13,11 +13,11 @@
...
       <article id="profile">
         {% if is_owner and profile.is_fxa_migrated %}
-        <h1 class="sumo-page-heading">{{ _('Account') }}</h1>
+        <h1 class="sumo-page-heading">{{ _('Your Account') }}</h1>
         <p class="mb-0"><strong>{{ user.email }}</strong></p>
         <p>
-          {{ _('You can change your email address on Firefox Accounts.') }}
-          <a href="https://support.mozilla.org/en-US/kb/change-primary-email-address-firefox-accounts">{{ _('Need help?') }}</a>
+          {{ _('To change your email, visit the Firefox Accounts page,') }}
+          <a href="https://support.mozilla.org/en-US/kb/change-primary-email-address-firefox-accounts">{{ _('learn more') }}</a>
```
[the remark this answers is not in the document]
Comment (LeoMcA) [reply kitsune/users/jinja2/users/profile.html:20]: There's also the same (old) wording in `edit_profile.html` which should be updated to be consistent.

File: kitsune/users/jinja2/users/profile.html [hunk 3]
```diff
@@ -32,11 +32,18 @@ <h1 class="sumo-page-heading">{{ _('Account') }}</h1>
           {% endif %}
           {{ flag_form(url('users.flag', object_id=profile.pk), False) }}
         {% endif %}
-        <h1 class="sumo-page-heading user">{{ display_name(profile.user) }}
+        <p>
+          {% trans %}
+          Your privacy is important to us, your username is always visible to the public,
+          if you would like to edit your username
+          {% endtrans %}
+          <a href="{{ url('users.edit_profile', profile.user.username) }}">{{ _('click here') }}</a>
```
[the remark this answers is not in the document]
Comment (LeoMcA) [reply kitsune/users/jinja2/users/profile.html:40]: This block should also be moved below the display name header

File: kitsune/users/jinja2/users/edit_profile.html [hunk 4]
```diff
@@ -51,28 +53,9 @@ <h1 class="sumo-page-heading">{{ title }}</h1>
...
-                  </p>
-                  {% endtrans %}
-                </span>
-              </a>
-              {% if profile.avatar %}
-                <a class="remove-avatar" href="{{ url('users.delete_avatar') }}">{{ _('Delete') }}</a>
-              {% endif %}
-            {% endif %}
+            <label>Avatar
+              <img class="avatar" src="{{ profile_avatar(profile.user) }}" alt="">
```
Comment (LeoMcA) [inline kitsune/users/jinja2/users/edit_profile.html:57]: IMO we need a bit of spacing between the label and img, looks a bit cramped currently.

File: kitsune/users/jinja2/users/edit_profile.html [hunk 5]
```diff
@@ -41,8 +43,8 @@ <h1 class="sumo-page-heading">{{ _('Account') }}</h1>
   <h1 class="sumo-page-heading">{{ title }}</h1>
```
Comment (LeoMcA) [inline kitsune/users/jinja2/users/edit_profile.html:43]: I think this should be a `<h2>` as well

File: kitsune/users/jinja2/users/profile.html [hunk 6]
```diff
@@ -13,11 +13,11 @@
 {% block content %}
       <article id="profile">
         {% if is_owner and profile.is_fxa_migrated %}
-        <h1 class="sumo-page-heading">{{ _('Account') }}</h1>
+        <h1 class="sumo-page-heading">{{ _('Your Account') }}</h1>
```
Comment (LeoMcA) [inline kitsune/users/jinja2/users/profile.html:16]: Also needs to be updated in `edit_profile.html`

File: kitsune/users/jinja2/users/edit_profile.html [hunk 7]
```diff
@@ -95,9 +78,9 @@ <h1 class="sumo-page-heading">{{ title }}</h1>
     {{ field(profile_form.locale) }}
     {{ field(profile_form.involved_from) }}
 
-    <div class="sumo-button-wrap reverse-on-desktop align-full">
-      <button class="sumo-button primary-button button-lg" type="submit">{{ _('Update My Profile') }}</button>
-      <button class="sumo-button button-lg" type="reset">Cancel</button>
+    <div class="sumo-button-wrap reverse-on-desktop align-right">
```
Comment (LeoMcA) [inline kitsune/users/jinja2/users/edit_profile.html:81]: `align-right` class isn't needed (it doesn't exist)

File: kitsune/sumo/static/sumo/scss/base/forms/_fields.scss (hunk 1 above)
Comment (LeoMcA) [reply kitsune/sumo/static/sumo/scss/base/forms/_fields.scss:359]: We can just change the text-align instead, and it needs to be wrapped in a `@include p.bidi()` to align left for rtl locales

**Review by LeoMcA — APPROVED** [review]
r+wc

would also change the commit message to clarify these are improvements to, rather than the commit implementing editable usernames

**Pull request merged**